submissions/description/cross_site_scripting_xss/cookie_based/template.md

```diff
@@ -0,0 +1,30 @@
+# Cookie-Based Cross-Site Scripting
+
+## Overview of the Vulnerability
+
+Cross-Site Scripting (XSS) is a type of injection attack where malicious JavaScript is injected into a website. When a user visits the affected web page, the Javascript executes within that user’s browser in the context of the domain. Cookie-based XSS can be found on this domain which allows an attacker to control code that is executed within a user’s browser. Additionally, the attacker needs to socially engineer the user to add the cookie containing malicious JavaScript into the user’s browser. If successful, the JavaScript will execute within that user’s browser in the context of this domain.
```

Comment:
Expanding upon this, you also need to add that this can be chained and is not just social engineering, I recommend reviewing CLRF attacks for Set Cookie injections in HTTP Responses.

Reply:
I've attempted to address a chained attack with CRLF in the latest commit